ddtrace/contrib/mariadb/patch.py

```diff
@@ -0,0 +1,54 @@
+import mariadb
+from ddtrace import Pin
+from ddtrace import config
+from ddtrace.contrib.dbapi import TracedConnection
+from ddtrace.vendor import wrapt
+from ddtrace.utils import get_argument_value
+
+from ...ext import db
+from ...ext import net
+
+from ...utils.formats import asbool
+from ...utils.formats import get_env
+
+
+config._add(
+    "mariadb",
+    dict(
+        trace_fetch_methods=asbool(get_env("mariadb", "trace_fetch_methods", default=False)),
+        _default_service="mariadb",
+    ),
+)
+
+
+def patch():
+    if getattr(mariadb, "_datadog_patch", False):
+        return
+    setattr(mariadb, "_datadog_patch", True)
+    wrapt.wrap_function_wrapper("mariadb", "connect", _connect)
+
+
+def unpatch():
+    if getattr(mariadb, "_datadog_patch", False):
+        setattr(mariadb, "_datadog_patch", False)
+        if isinstance(mariadb.connect, wrapt.ObjectProxy):
+            mariadb.connect = mariadb.connect.__wrapped__
+            if hasattr(mariadb, "Connect"):
+                mariadb.Connect = mariadb.connect
```

Comment:
I don't see where we patch `mariadb.Connect` - we only patch `mariadb.connect`. Should we be patching `Connect` above?

Reply:
Good catch, Connect is not an exposed function